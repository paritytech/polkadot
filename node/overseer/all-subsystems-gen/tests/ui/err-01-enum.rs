#![allow(dead_code)]

use polkadot_overseer_all_subsystems_gen::AllSubsystemsGen;

#[derive(Clone, AllSubsystemsGen)]
enum AllSubsystems<A,B> {
	A(A),
	B(B),
}

fn main() {
	let all = AllSubsystems::<u8,u16>::A(0u8);
}
