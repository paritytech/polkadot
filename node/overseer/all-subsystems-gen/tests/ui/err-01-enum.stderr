error: expected `struct`
 --> $DIR/err-01-enum.rs:6:1
  |
6 | enum AllSubsystems<A,B> {
  | ^^^^
